### DISCUSSION TITLE
What is the purpose of start % and stop % values?

### DISCUSSION THREAD

**Discussion by markandgo** [Rem0o/FanControl.Releases#1330] [Q&A] [answered]
I know that the values are supposed to represent when a fan turns on from idle and off from spinning. Are they for our own reference or do these values affect anything? Thanks!

**Comment by Rem0o** [comment, marked as answer]
They do affect stuff.

Any fan curve that hits below a stop% on a specific control will get snapped to 0.  You don't want to run a voltage/power to a fan where it's not enough to make it spin.

As for start%, let say your start is 50 and your stop is 30.  
A fan curve asks for 40%, and the fan is currently not spinning.   The fan will get spun up to 50%, wait a bit until it spins properly, then will get drop down to 40. It's used as a "kick off", kinda like when you start your car up the engine will rev up a bit over the idle speed.

**Comment by phantum29** [comment]
Would it be possible to then adjust the 0 - 100% range for the fan to the values determined by the start / stop % control? This would make it a bit easier when making curves such that I want different fans using the same curve with their relative % at any given temp. Currently I need to have separate curves for each fan based around its start/top speed %. Thanks!

**Reply by Rem0o** [reply]
It could be done, but it would add yet another layer of complexity, and I'm not sure it would actually help you manage multiple different fans at once with the same curve.  

One thing you could do is to set your fan curve minimum to be the highest stop % of all your fans you want to use that curve, then do a hard cut-off to 0% at the temperature you want to shut them off.   You can always apply offsets to your controls too by the way, that's an existing way to deal with different types of fans.  If a fan has a stop% 5 above another fan, add a 5% offset on it.

**Reply by phantum29** [reply]
That's a good suggestion. I guess I didn't fully understand the offset either, thanks so much!
